types/codec.go

```diff
@@ -14,5 +14,5 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 
 // Register the sdk message type
 func RegisterInterfaces(registry types.InterfaceRegistry) {
-	registry.RegisterInterface("cosmos_sdk.v1.Msg", (*Msg)(nil))
+	registry.RegisterInterface("cosmos.Msg", (*Msg)(nil))
```

Comment:
Good point, I changed to `"cosmos.v1beta1.Msg"` for now, since everything is in beta. But yeah, I guess we should also define which interfaces are beta/stable, maybe add a line somewhere in #6954?